corehq/apps/receiverwrapper/views.py

```diff
@@ -81,38 +81,16 @@ def _process_form(request, domain, app_id, user_id, authenticated,
 
     with TimingContext() as timer:
         app_id, build_id = get_app_and_build_ids(domain, app_id)
-        submission_post = SubmissionPost(
-            instance=instance,
-            attachments=attachments,
-            domain=domain,
-            app_id=app_id,
-            build_id=build_id,
-            auth_context=auth_cls(
-                domain=domain,
-                user_id=user_id,
-                authenticated=authenticated,
-            ),
-            location=couchforms.get_location(request),
-            received_on=couchforms.get_received_on(request),
-            date_header=couchforms.get_date_header(request),
-            path=couchforms.get_path(request),
-            submit_ip=couchforms.get_submit_ip(request),
-            last_sync_token=couchforms.get_last_sync_token(request),
-            openrosa_headers=couchforms.get_openrosa_headers(request),
-            force_logs=request.GET.get('force_logs', 'false') == 'true',
-        )
+        submission_post = _submission_post_object(
+            instance, attachments, domain, app_id, build_id, auth_cls, user_id,
+            authenticated, request)
```

Comment:
Changing keyword arguments to a wrapper with positional arguments doesn't seem like a useful refactor to me. The problem here is that `SubmissionPost` takes too many arguments. In that case using keyword arguments is better because it ensures proper alignment if/when arguments are removed or (heaven forbid) added.

Reply:
https://github.com/dimagi/commcare-hq/pull/28424/commits/db845415073708287a6d9c4632f28d7ca84d4bd8

Reply:
My preference would be to remove https://github.com/dimagi/commcare-hq/pull/28424/commits/6606871ede2f456fd8f10966089d41e0cb97c503 (and https://github.com/dimagi/commcare-hq/commit/db845415073708287a6d9c4632f28d7ca84d4bd8) from this PR. In my opinion these changes add more code without substantially improving clarity or simplicity.

Reply:
Done, removed those commits.